JustSaying/AwsTools/MessageHandling/SnsTopicBase.cs

```diff
@@ -54,7 +54,11 @@ public async Task PublishAsync(Message message, CancellationToken cancellationTo
             try
             {
                 var response = await Client.PublishAsync(request, cancellationToken).ConfigureAwait(false);
-                _eventLog.LogInformation($"Published message: '{request.Subject}' with content {request.Message}");
+                _eventLog.LogInformation(
```

Comment:
It's probably best to leave this as non-structured for the v6 branch for consistency.

I don't know what would be better, but also as the content isn't escaped this might could out looking a bit messy.

Meh, I don't have strong opinions either way.

Reply:
I have used the structured version because I wanted to make sure the Request Id was always logged and not truncated because the logged message reaches the logging client limit. I will refactor it and put it in front to make sure it is always logged.

Reply:
Hmm, in that case let's leave it structured, otherwise it might break if anyone is grepping the logs.